bootstrap/src/main/java/com/splunk/opentelemetry/javaagent/ConfigurationConstants.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.javaagent;
```

Comment:
Why `javaagent`? This class is used outside of `javaagent` package

Reply:
Wat? The constants are used by the agent, why would this be an improper package? Why are you concerned that this is used outside the package? Do you have a suggestion for a better package?

Reply:
All classes that use this constant does not have `javaagent` in their package names, so it caught my eye. Yes, all classes in `bootstrap` module live in `com.splunk.opentelemetry.javaagent`, so may be it is OK.

But on yet another hand: why not put these constants in `SplunkConfiguration` class, which already has a couple of them?

Reply:
Yeah that works too. Moved them.